core/src/main/java/com/novoda/landingstrip/animation/FastOutSlowInInterpolator.java

```diff
@@ -0,0 +1,45 @@
+package com.novoda.landingstrip.animation;
+
+class FastOutSlowInInterpolator extends LookupTableInterpolator {
```

Comment:
to keep it as the original so that we can easily sync it if there are any changes